README.md

```diff
@@ -117,28 +117,29 @@ To use feature flags, check out the documentation in (the README of)
 
 ### Updating the index mapping
 
-Whenever the mappings JSON file is updated, a new version value should be written to the mapping
-metadata in `"_meta": {"version": VALUE}`.
+Whenever a mapping JSON file is updated, a new version value should be written to the mapping
+metadata in `"_meta": {"version": VALUE}`. Mapping files should be generated from framework data.
 
-Mapping can be updated by issuing a PUT request to the existing index enpoint:
+An index's mapping can be updated by issuing a PUT request to the existing index endpoint:
 
 ```
 PUT /g-cloud-index HTTP/1.1
 Authorization: Bearer myToken
 Content-Type: application/json
 
-{"type": "index"}
+{"type": "index", "mapping": "services"}
```

Comment:
So I think this could still go either way - we might up with a single index (and mapping) to cover services from other frameworks too. It's not impossible that's what "scaling" would look like, anyway ;-) For now, I'd rather just leave it.